### DISCUSSION TITLE
Ultimate Tex features within SSBH Editor

### DISCUSSION THREAD

**Discussion by zrksyd** [ScanMountGoat/ssbh_editor#144] [Ideas]
Is it possible to import / export NUTEXB textures within SSBH Editor?

**Comment by ScanMountGoat** [comment]
Is it possible to get the code working with SSBH Editor? Definitely. Is it something I plan on implementing? I would need to think about it some more. Is there a use case that you think would work better with SSBH Editor compared to Ultimate Tex? 

The main difference is that Ultimate Tex uses CPU encoding and decoding. This means the CLI version of Ultimate Tex can run places where SSBH Editor can't like virtual machines or machines without a GPU. SSBH Editor takes advantage of GPU hardware support for reading compressed texture data. Ultimate Tex would only be necessary for the encoding portion.

Another difference is that SSBH Editor is designed to work with one file at a time within a folder rather than groups of files. I'm not sure what the advantages of working with SSBH Editor for textures would be compared to Switch Toolbox for people running Windows. SSBH Editor also focuses exclusively on model files, which doesn't include bntx.
